### PR TITLE
ui: Rename "private messages" / "PMs" to "direct messages" / "DMs", etc.

### PR DESCRIPTION
**Pull request by chrisbobbe**
As mentioned in the issue, there are various variable names that still use "private messages"; we're leaving those be for now.

PR #5728 is open for a related API migration: to use 'direct' instead of 'private' in POST /messages, when supported.

Fixes: #5731

### CODE REVIEW

File: src/settings/SettingsScreen.js [hunk 1]
```diff
@@ -82,7 +82,7 @@ export default function SettingsScreen(props: Props): Node {
             key: 'conversation-views-only',
             title: 'Only in conversation views',
             subtitle:
-              'Messages will be automatically marked as read only when viewing a single topic or private message conversation.',
+              'Messages will be automatically marked as read only when viewing a single-topic or direct-message conversation.',
```
Comment (gnprice) [inline src/settings/SettingsScreen.js:85]: This revised text suggests these are two kinds of conversations, and there are other kinds of conversations. I don't think that's a way we normally use "conversation".

The original text was intended to mean "viewing a single topic (as opposed to a whole stream), or viewing a (single) PM conversation". Could perhaps be clarified by adding one hyphen: "direct-message conversation" (or previously could have been "private-message conversation"). Or could expand it more wordily if that doesn't seem clear.

In the interest of getting this rename in, probably best to minimize the other wording changes and leave those for a possible followup.

**Review by gnprice — COMMENTED** [review]
Thanks! Looks good except one comment where there were additional changes.

File: src/settings/SettingsScreen.js (hunk 1 above)
Comment (chrisbobbe) [reply src/settings/SettingsScreen.js:85]: Ah OK, right! Thanks for the catch; I'll make it so "private" to "direct" is the only change here.

**Comment by chrisbobbe** [comment]
Thanks for the review! Merged, with that fix.

**Pull request merged**